internal/provider/waypoint/resource_waypoint_add_on_definition.go

```diff
@@ -109,6 +109,10 @@ func (r *AddOnDefinitionResource) Schema(ctx context.Context, req resource.Schem
 				Description: "The markdown template for the Add-on Definition README (markdown format supported).",
 				Optional:    true,
 			},
+			"terraform_project_id": schema.StringAttribute{
+				Required:    true,
+				Description: "The ID of the Terraform Cloud Project to create workspaces in",
```

Comment:
I imagine this isn't something we can change trivially, right? We couldn't just change the project ID value to another project, that wouldn't move the workspace in TFC to the new project, correct? Assuming that's the case, we should mark this as something that needs to trigger a replacement if changed as part of the `schema.StringAttribute`:

```go
  PlanModifiers: []planmodifier.String{
    stringplanmodifier.RequiresReplace(),
  },
```

Reply:
That makes a lot of sense to me!